Prog/Voices_q15_recode.sas

```diff
@@ -55,73 +55,326 @@ data Q15_recode_unq;
-  format income income.;
-
-  age =.;
-  IF (18 <= PPAGE <= 34) THEN age = 1;
-  IF (35 <= PPAGE <= 64) THEN age = 2;
-  IF (PPAGE >= 65) THEN age = 3;
-  format age age.;
-
-  gender =.;
-  IF (PPGENDER=1) THEN gender = 0;
-  IF (PPGENDER=2) THEN gender = 1;
-  format gender gender.;
-
-  homeown =.;
-  IF (PPRENT=2) or (PPRENT=3)  THEN homeown = 0;
-  IF (PPRENT=1) THEN homeown = 1;
-  format homeown homeown.;
-
-  label
-    race = "Respondent's Race/Ethnicity"
-    educ = "Respondent's educational attainment"
-    income = "Respondent's HH income"
-	age = "Respondent's age"
-    gender = "Respondent's gender"
-    homeown = "Respondent's homeownership status";
-
   rename recode=Q15_recode;
   
 run;
 
+proc transpose data=Voices_Q15_notyetrecoded out=Voices_Q15_notyetrecoded_1;
+
+	by caseid;
+	var Q15_Text1 Q15_Text2 Q15_Text3; 
+
+run;
+data Voices_Q15_notyetrecoded_2 (drop=_name_);
+
+	set Voices_Q15_notyetrecoded_1 (drop=_label_);
+
+	respnum=.;
+	if _name_="Q15_Text1" then respnum=1;
+	if _name_="Q15_Text2" then respnum=2;
+	if _name_="Q15_Text3" then respnum=3;
+
+run; 
+data Voices_Q15_toreview;
+
+	set Voices_Q15_r1 (rename=(Q15_Text1=Q15_Text))
+		Voices_Q15_r2 (rename=(Q15_Text2=Q15_Text))
+		Voices_Q15_r3 (rename=(Q15_Text3=Q15_Text))
+		Voices_Q15_notyetrecoded_2(rename=(col1=Q15_Text));
+
+		Q15_text=propcase(Q15_text);
+
+		drop fuzzratio;
+run;
+%let list='ENTERTAINMENT' 'FOOD' 'CULTURE' 'MUSEUMS' 'MONUMENTS' 'PERFORM' 'ARTS' 'SPORTS' 'NIGHTLIFE' 'JOBS' 'PEOPLE' 'LOCATION' 'DIVERSITY' 'TRANSPORTATION' 'BUS' 'METRO'
+    'AIRPORTS' 'NATURE' 'COMMUNITY' 'SCHOOLS' 'WEATHER' 'LIFE' 'FAMILY' 'SHOPPING' 'SERVICES' 'CAPITAL' 'ECONOMY' 'NOTHING' 'SAFETY' 'HOUSING' 'DC' 'POLITICS' 'WALKABILITY'
+    'COSTS' '';
+
+%macro printlist;
+
+%do i=1 %to 35;
+%let var=%scan(&list,&i.," ");
+
+proc freq data=Voices_Q15_toreview;
+tables Q15_Text ;
+where  Q15_recode=&var.;
+Title "Coded as &var"; 
+
+run;
+%end; 
+
+%mend;
+
+%printlist;
+
+data Voices_Q15_newrecode (drop=diversity culture politics capital monuments transpo metro jobs food nightlife entertain services people);
+
+	set Voices_Q15_toreview;
+
+			culture=index(Q15_text,"Cultural Activ");
+			culture=culture+index(Q15_text,"Cultural Events");
+			culture=culture+index(Q15_text,"Cultural Sites");
+			culture=culture+index(Q15_text,"Cultural Places");
+			culture=culture+index(Q15_text,"Cultyural Events");
+			culture=culture+index(Q15_text,"Cutural Events");
+			culture=culture+index(Q15_text,"Access To Cultural Institutions"); *was in location;
+			culture=culture+index(Q15_text,"Access To Cultural/Historical Resources");*was in location;
+			culture=culture+index(Q15_text,"Access To Free Cultural");*was in location;
+			culture=culture+index(Q15_text,"Cultural Access");*was in location;
+			culture=culture+index(Q15_text,"Proximity To Cultural Institutions");*was in location;
+			culture=culture+index(Q15_text,"Proximity To Cultural And Historic");*was in location;
+
```

Comment:
Seems like these should also be categorized as culture (currently under entertainment, monuments).
```
Cultural & Social Events
Cultural Activites
Cultural Activities
Cultural Activities, Museums, Etc.
Cultural Activitites
Cultural Events
Cultural Events And Activities
Cultural Events And Museums
Cultural Places And Activities
Cultural Sites And Events
Cultyural Events
Cutural Events
Diverse Selection Of Cultural Activities
Easy Access To Cultural Events
Lots Of Cultural Activities
Lots Of Cultural Activities Available
Lots Of Cultural Events
Lots Of Cultural Events, Museums
Plenty Of Cultural Activities, Restaurants
Proximity To Cultural Activities
Variety Of Cultural Activities
Proximity To Cultural Sights
```
And these under museums.
```
Museums & Activities
Museums, Other Activities
Museums/Cultural Activities
```